### ISSUE TITLE
functools.partial error in version 3.0.0

### ISSUE THREAD

**Issue by sjktje** [funkybob/django-classy-settings#38]
```python
denv = env(prefix='DJANGO_')

class Settings(BaseSettings):
    DEBUG = denv.bool(True)

```

raises error:

AttributeError: 'functools.partial' object has no attribute 'bool'

**Comment by sjktje** [comment]
Also, trying to access an environment variable like this does not work:

```python
class Settings(BaseSettings):
    SECRET_KEY = env('insecure-key', prefix='DJANGO_')

```
The above throws an exception, even though DJANGO_SECRET_KEY is set in the environment:

`django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.`

**Comment by funkybob** [comment]
Thanks for the heads up.

These problems are fixed in release 3.0.1!
